### ISSUE TITLE
Pre-prepared cache data for VidOR

### ISSUE THREAD

**Issue by Chango-G** [Dawn-LX/VidSGG-BIG#21]
您好，请问能提供一下数据集VidOR的pre-prepared cache data的百度云或者其他的下载方式吗？由于文件太大MEGA上的数据无法下载完成，谢谢。
![cache](https://github.com/Dawn-LX/VidSGG-BIG/assets/51011605/d5dd6a86-3ec9-44ae-8c6c-6e6d7577c8e2)

**Comment by Dawn-LX** [comment]
> 您好，请问能提供一下数据集VidOR的pre-prepared cache data的百度云或者其他的下载方式吗？由于文件太大MEGA上的数据无法下载完成，谢谢。 ![cache](https://user-images.githubusercontent.com/51011605/269295937-d5dd6a86-3ec9-44ae-8c6c-6e6d7577c8e2.png)

没钱开百度云会员，百度云我要传一百年。。。。

然后，这个cache data 我那时候直接存成object了，当时刚学pytorch不太懂，搞了很多很傻的操作。现在这堆屎山代码看了就头疼。

**Comment by Chango-G** [comment]
> > 您好，请问能提供一下数据集VidOR的pre-prepared cache data的百度云或者其他的下载方式吗？由于文件太大MEGA上的数据无法下载完成，谢谢。 ![cache](https://user-images.githubusercontent.com/51011605/269295937-d5dd6a86-3ec9-44ae-8c6c-6e6d7577c8e2.png)
> 
> 没钱开百度云会员，百度云我要传一百年。。。。
> 
> 然后，这个cache data 我那时候直接存成object了，当时刚学pytorch不太懂，搞了很多很傻的操作。现在这堆屎山代码看了就头疼。

hhh好吧，我再试试从MEGA上下载吧，谢谢啦。

**Comment by Chango-G** [comment]
打扰一下，还有一个问题：论文中Classification Head中分类predicate category时，公式中使用了Softmax，但是在代码中`pred_logits = pred_logits + pred_bias`并没有看到使用，请问是在别的地方实现了吗？
![softmax](https://github.com/Dawn-LX/VidSGG-BIG/assets/51011605/7f9cd006-11d1-4a73-9d62-d0a5fff85c00)

**Comment by Dawn-LX** [comment]
> 文中Classification He

softmax 是在分类层做的， 这里的代码只对应括号内的相加。
